cluster-autoscaler/provisioningrequest/checkcapacity/orchestrator.go

```diff
@@ -151,10 +152,10 @@ func (o *provReqOrchestrator) scaleUp(unschedulablePods []*apiv1.Pod) (bool, err
 	}
 	st, _, err := o.injector.TrySchedulePods(o.context.ClusterSnapshot, unschedulablePods, scheduling.ScheduleAnywhere, true)
 	if len(st) < len(unschedulablePods) || err != nil {
-		setCondition(provReq, v1beta1.Provisioned, metav1.ConditionFalse, CapacityIsFoundReason, "Capacity is not found, CA will try to find it later.", metav1.Now())
+		conditions.AddOrUpdateCondition(provReq, v1beta1.Provisioned, metav1.ConditionFalse, conditions.CapacityIsFoundReason, "Capacity is not found, CA will try to find it later.", metav1.Now())
```

Comment:
Maybe I'm missing something, byt the Condition Reason and Mesg don't seem to match.
`CapacityIsFoundReason` --> `Capacity is not found, CA will try to find it later. `
Is this expected?